compiler/include/ParamForLoop.h

```diff
@@ -48,22 +48,22 @@ class ParamForLoop : public LoopStmt
                                       LabelSymbol* continueLabel,
                                       LabelSymbol* breakLabel,
                                       BlockStmt*   initBody);
-  virtual               ~ParamForLoop();
+                        ~ParamForLoop();
```

Comment:
`~ParamForLoop() override;`

Repeat for all of the destructors. I've only listed a few of them. `override` works for destructors by ensuring that the base class destructor is `virtual`.

Reply:
Good point.

I am leaving this for future work so I can make progress with the rest of this PR.